server/routes/github/github_user_repos.py

```
@@ -140,7 +145,7 @@ def get_user_branches(user_id, repo_full_name):
         
         while True:
             response = requests.get(
```

Ruff (0.15.10)

[error] Probable use of `requests` call without timeout

(S113)